about-page.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en-us">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Go Green - Summit 2022</title>
+    <meta name="description" content="About Go Green - Summit 2022">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="stylesheets/styles.css">
+</head>
+
+<body class="about">
+    <section class="top-section">
+        <div class="top-cover">
+            <nav class="inactive">
```

Comment:
- [ ] The current font you have on the entire page is `Arial`. you would need to use `Lato` & `COCOGOOSE` fonts in order to match the fonts of the reference design. I can see that you have made an attempt to use these in the CSS but unfortunately, these CSS fonts do not show up on the final page. please check your CSS and fonts and make sure that the fonts that will show up on the page are the correct ones.